### ISSUE TITLE
Changing the review status of multiple bugs at once

### ISSUE THREAD

**Issue by whisperity** [Ericsson/codechecker#1353]
This might be **very** low priority, but if a source file has 70 of the similar report (and all of them is a valid finding) manually clicking `Confirmed bug` on 70 instances is very tedious.

**Comment by csordasmarton** [comment]
@whisperity We change the review status by bug hash, so if you have 70 similar reports, it will "change" the review status of all the similar reports. Did you update the page (Ctrl+F5)?

**Comment by whisperity** [comment]
Yes. These bugs are not similar in the sense of they have the same bughash. Actually, they really shouldn't.
They are just similar in the sense that they come from what appears to be generated code, and they are centered around roughly the same issue. Based on uniqueing, they are uniqued out into like... 22 bugs or so.
